src/app/services/model/chart.modal.ts

```diff
@@ -0,0 +1,11 @@
+interface AdditionalData {
```

Comment:
if this interface you need for Chart, let's also add Chart in naming: ChartAddionalData, so when it is met in the code each can understand where it belongs to